WebSocketClient/script.js

```diff
@@ -0,0 +1,50 @@
+(function () {
+    "use strict";
+    
+    const URL = "ws://localhost:5000/ws";
+
+    const connect = () => {
+        let socket = new WebSocket(URL);
+        socket.onopen = (event) => {
+            console.log("OPEN");
+        };
+        socket.onclose = (event) => {
+            console.log("CLOSE");
+        };
+        socket.onerror = (error) => {
+            console.error(error.message);
+        };
+        socket.onmessage = (event) => {
+            const message = event.data;
+            const list_messages = document.getElementById("list-messages");
+            let li = document.createElement("li");
+            li.appendChild(document.createTextNode(message));
+            list_messages.appendChild(li);
+        };
+        return socket;
+    }
+
+    const onSubmitForm = (socket) => (
```

Comment:
Może się czepiam, ale definiując w ten sposób to jest widoczne w pliku tylko poniżej definicji. Przy użyciu `function` byłaby funkcja zdefiniowana globalnie

Reply:
Czekaj, jest widoczna tak samo jak `function`, przed definicja tez ja widac.